cp2k/docker/spack.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+source /opt/spack/share/spack/setup-env.sh
+
+spack env activate /opt/cp2k_environment
+spack external find --all
+spack concretize -f
+spack install -j16
+ln -s `spack find -p cp2k@master | tail -n 1 | awk -e 'BEGIN{FS=" "} {print $2}'` /opt/cp2k
```

Comment:
Building the docker container failed at this line, and removing the `-e` option from the awk command seems to fix the problem. Can this be done?
-ln -s `spack find -p cp2k@master | tail -n 1 | awk -e 'BEGIN{FS=" "} {print $2}'` /opt/cp2k
+ln -s `spack find -p cp2k@master | tail -n 1 | awk 'BEGIN{FS=" "} {print $2}'` /opt/cp2k